jobs/rocoto/gempak.sh

```diff
@@ -12,14 +12,18 @@ IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
 
 export FHR3 jobid
 for fhr in "${fhr_list[@]}"; do
-	FHR3=$(printf '%03d' "${fhr}")
-	jobid="${job}_f${FHR3}.$$"
-	###############################################################
-	# Execute the JJOB
-	###############################################################
-	"${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
-	status=$?
-	[[ ${status} -ne 0 ]] && exit "${status}"
+    FHR3=$(printf '%03d' "${fhr}")
+    jobid="${job}_f${FHR3}.$$"
+    ###############################################################
+    # Execute the JJOB
+    ###############################################################
+    if [[ "${RUN}" == "gefs" ]]; then
+        echo "add J-script here for GEFS."
+    else
+        "${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
+        status=$?
+        [[ ${status} -ne 0 ]] && exit "${status}"
```

Comment:
While we're in here, please change the following to match similar changes happening in other scripts:
```suggestion
        err=$?
        if [[ ${err} -ne 0 ]]: then
          exit "${err}"
        fi
```